lib/player.js

```diff
@@ -3777,7 +3859,8 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
   }
 
   /**
-   * Get the TextTracks with the 'metadata' kind.
+   * Ignore the TextTracks with the 'metadata' or 'chapters' kind, or the one
```

Comment:
This comment doesn't makes sense with the code.  It looks like you're only returning 'metadata' tracks here.

Reply:
Sorry I already changed it, it was a mistake due to copy-paste